File: leetcode3/염혜정/3813. Vowel-Consonant Score.java
// 97 ~ 122
// o(n)

class Solution {
    public int vowelConsonantScore(String s) {
        Set<Character> vowels = new HashSet<>(List.of('a', 'e', 'i', 'o', 'u'));

        int v = 0;
        int c = 0;
        for (char letter : s.toCharArray()) {
            int askii = (int) letter;
            if (askii < 97 || askii > 122) continue;

            if (vowels.contains(letter)) v++;
            else c++;
        }
        if (c==0) return 0;
        return v / c;
    }
}

File: leetcode3/남효정/3813. Vowel-Consonant Score.py
class Solution:
    def vowelConsonantScore(self, s: str) -> int:
        v = 0
        vowels = ['a', 'e', 'i', 'o', 'u']
        for vowel in vowels:
            v += s.count(vowel)
        return v // (len(s) - v)